apps/web/src/pages/admin/meeting-room/index.tsx

```
@@ -2,58 +1,27 @@
+import { Resource } from "@repo/constants/type";
+import { getGroupedResourceListBySubtype } from "@repo/lib/api/resourceUtils";
 import Button from "@src/components/commons/Button";
-import ListItem from "@src/components/commons/ListItem";
-import Skeleton from "@src/components/commons/Skeleton";
+import ErrorBoundary from "@src/components/commons/ErrorBoundary";
 import useModal from "@src/hooks/useModal";
-import { useQuery } from "@tanstack/react-query";
-import { useMemo } from "react";
+import { useGetReservations } from "@src/pages/meeting-rooms/hooks/useGetReservations";
+import Error from "@src/pages/seats/Error";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)